merge_requests_test.go

```diff
@@ -141,6 +141,7 @@ func TestGetMergeRequest(t *testing.T) {
 	mrUpdate := time.Date(2019, 8, 20, 9, 9, 56, 690000000, time.UTC)
 	require.Equal(t, mergeRequest.UpdatedAt, &mrUpdate)
 	require.Equal(t, mergeRequest.HasConflicts, true)
+	require.Equal(t, mergeRequest.FirstContribution, true)
```

Comment:
Please swap this line with the one above for consistent ordering.

Reply:
done